processing/src/test/java/org/apache/druid/indexer/TaskLocationTest.java

```diff
@@ -54,6 +54,18 @@ public void testTlsForPeonJobs()
     Assert.assertEquals(2, tls.getTlsPort());
   }
 
+  @Test
+  public void testDefaultK8sJobName()
+  {
+    TaskLocation noK8sJobName = TaskLocation.create("foo", 1, 2, false);
+    Assert.assertNull(noK8sJobName.getK8sPodName());
+    noK8sJobName = TaskLocation.create("foo", 1, 2);
+    Assert.assertNull(noK8sJobName.getK8sPodName());
+
+    TaskLocation k8sJobName = TaskLocation.create("foo", 1, 2, false, "job-name");
+    Assert.assertEquals("job-name", k8sJobName.getK8sPodName());
```

Comment:
nit: it would be better to split this test into 2

Reply:
k8sJobName != k8sPodName, in this case i think the pod name is more useful but we could add both if needed?

Reply:
fixed the tests